core/app/models/spree/payment.rb

```diff
@@ -9,6 +9,7 @@ class Payment < Spree::Base
 
     NON_RISKY_AVS_CODES = ['B', 'D', 'H', 'J', 'M', 'Q', 'T', 'V', 'X', 'Y'].freeze
     RISKY_AVS_CODES     = ['A', 'C', 'E', 'F', 'G', 'I', 'K', 'L', 'N', 'O', 'P', 'R', 'S', 'U', 'W', 'Z'].freeze
+    INOPERATIVE_STATES  = %w(failed invalid)
```

Comment:
@nishant-cyro In my views Unsuccessful is more appropriate to INOPERATIVE. INOPERATIVE is like something which is not in use.